bundles/c/c_lexer.moon

```diff
@@ -62,7 +55,6 @@ howl.aux.lpeg_lexer ->
 
   include_stmt = sequence {
     c('preproc', '#include'),
-    ws^1,
```

Comment:
I see. The problem is now only include statements without white space are handled correctly. `ws^0` is probably what you want in this case. Similar to my other comment, could you add an additional example to (and verify against) the example.c file?